tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -62,8 +62,12 @@ public void RunAllTests()
                 AcquireTokenSilentTest,
 
                 AcquireTokenADFSV3InteractiveFederatedTest,
+                AcquireTokenADFSV3InteractiveManagedTest,
+                AcquireTokenMsaUserTest,
                 AcquireTokenADFSV4InteractiveFederatedTest,
+                AcquireTokenADFSV4InteractiveManagedTest,
```

Comment:
there is no such thing as adfsv4 managed

Reply:
same comment as above